<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>containingWidget</class>
 <widget class="QWidget" name="containingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="unseededCheckBox">
         <property name="text">
          <string>Unseeded</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="smoothingLabel">
         <property name="text">
          <string>Smoothing:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="smoothingComboBox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="smoothingDoubleSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="computeLabel">
         <property name="text">
          <string>Compute Minima from:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="computeComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate Seeds</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
